lua/grapple/tags.lua

```diff
@@ -30,7 +30,7 @@ end
 ---@param key Grapple.TagKey
 ---@return Grapple.Tag
 local function _get(scope, key)
-    return _scoped_tags(scope)[key]
+    return vim.tbl_extend("keep", _scoped_tags(scope)[key], { key = key })
```

Comment:
I fell into a trap ! The typing `Grapple.Tag` was not actually respected beforehand.